samples/nrf5340/multicore/aci/CMakeLists.txt

```diff
@@ -12,6 +12,6 @@ if (CONFIG_INCLUDE_NET_CORE_IMAGE)
     NAME cpunet
     SOURCE_DIR ${CMAKE_CURRENT_LIST_DIR}/../cpunet
     DOMAIN cpunet
```

Comment:
Also shouldnt we change here all cpunet to remote?

Reply:
I am not sure about it. This sample is directly NRF5340 sample. Lets rename and align it if it would be changed to more general form.